main.py

```diff
@@ -29,25 +30,38 @@ def process_email(email_path):
         elif k.lower() == "n":
             logging.info("---------Stop processing emails---------")
             return
+    email_counter = 0  
```

Comment:
We don't need `email_counter`, we can get this info in line 34, for example, use `for i in range(len(mail_info))`